src/estimagic/visualization/convergence_plot.py

```diff
@@ -30,6 +24,9 @@ def convergence_plot(
     stopping_criterion="y",
     x_precision=1e-4,
     y_precision=1e-4,
+    combine_plots_in_grid=True,
+    template=PLOTLY_TEMPLATE,
+    palette=px.colors.qualitative.Plotly,
```

Comment:
For the color palette we should also have a global (imported from config) as default. I.e. similar to `PLOTLY_TEMPLATE`.

While we are at it: I think the colors look really bad in derivative plots and not too good in some others. Maybe we should try out D3, G10 and T10 from [here](https://plotly.com/python/discrete-color/) to see if we can get a nicer default without much effort.

Reply:
The Set2 template provides a slightly better default, but prospectively we may want to create our own.

Reply:
My suggestion: Let's not get distracted. Economists should trust specialization. If anything, pick some color cycle from somewhere, but let's not take on the burden of discussions about this sort of thing.